pywbemtools/pywbemcli/_connection_repository.py

```diff
@@ -337,12 +444,10 @@ def delete(self, name):
             Name of the connection to delete
 
         Raises:
-            KeyError: No server definition exists in the connections file
-            with this name
-
-            ValueError: Connection file is not valid YAML
-
-            IOError: The connections file does not exist.
+            ConnectionFileLoadError: if it loads the file and the file is
+            invalid.
+            KeyError: if name does not exist in the file
+            ConnectionFileWriteError: If the file cannot be written
```

Comment:
Missing `s` in exception class names: `ConnectionFile...` -> `ConnectionsFile...`